rest-server/src/main/resources/nubesgen/bicep/modules/spring-cloud/spring-cloud.bicep.mustache

```diff
@@ -0,0 +1,78 @@
+// Azure Spring Cloud - Bicep module
+// Generated by NubesGen (www.nubesgen.com)
+@description('The name of your application')
+param applicationName string
+
+@description('The environment (dev, test, prod, ...')
+@maxLength(4)
+param environment string = 'dev'
+
+@description('The number of this specific instance')
+@maxLength(3)
+param instanceNumber string = '001'
+
+@description('The Azure region where all resources in this example should be created')
+param location string
+
+@description('An array of NameValues that needs to be added as environment variables')
+param environmentVariables object
+
+@description('A list of tags to apply to the resources')
+param resourceTags object
+
+{{#applicationTierBasic}}
+var springCloudSkuName = 'B0'
+var springCloudSkuTier = 'Basic'
+{{/applicationTierBasic}}
+{{#applicationTierStandard}}
+var springCloudSkuName = 'S0'
+var springCloudSkuTier = 'Standard'
+{{/applicationTierStandard}}
+
+resource springCloudService 'Microsoft.AppPlatform/Spring@2020-07-01' = {
+  name: 'asc-${applicationName}-${environment}'
+  location: location
+  tags: resourceTags
+  sku: {
+    
+    name: springCloudSkuName
+    tier: springCloudSkuTier
+  }
+}
+
+resource springCloudApp 'Microsoft.AppPlatform/Spring/apps@2021-06-01-preview' = {
+  name: 'app-${applicationName}-${environment}-${instanceNumber}'
+  parent: springCloudService
+  identity: {
+    type: 'SystemAssigned'
+  }
+  properties: {
+    public: true
+  }
+}
+
+resource springCloudAppDeployment 'Microsoft.AppPlatform/Spring/apps/deployments@2021-06-01-preview' = {
+  name: 'default'
+  parent: springCloudApp
+  sku: {
+    capacity: 1
+    name: springCloudSkuName
+    tier: springCloudSkuTier
+  }
+  properties: {
+    source: {
+      type: 'Jar'
+      relativePath: '<default>'
+    }
+    deploymentSettings: {
+      resourceRequests: {
+        cpu: '1'
+        memory: '1Gi'
+      }
+      runtimeVersion: 'Java_11'
+      environmentVariables: environmentVariables
+    }
+  }
+}
+
+output application_hostname string = springCloudApp.properties.url
```

Comment:
I think this doesn't work (the output).
I've tried a deployment with **Standard** SKU and I got the following error: 
```json
{"status":"Failed","error":{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"Conflict","message":"{  \"status\": \"Failed\",  \"error\": {    \"code\": \"ResourceDeploymentFailure\",    \"message\": \"The resource operation completed with terminal provisioning state 'Failed'.\",    \"details\": [      {        \"code\": \"DeploymentOutputEvaluationFailed\",        \"message\": \"Unable to evaluate template outputs: 'application_hostname'. Please see error details and deployment operations. Please see https://aka.ms/arm-debug for usage details.\",        \"details\": [          {            \"code\": \"DeploymentOutputEvaluationFailed\",            \"target\": \"application_hostname\",            \"message\": \"The template output 'application_hostname' is not valid: The language expression property 'url' doesn't exist, available properties are 'public, provisioningState, fqdn, httpsOnly, createdTime, temporaryDisk, persistentDisk, enableEndToEndTLS'..\"          }        ]      }    ]  }}"}]}}
```

If I replace the last line by the following, it works.
```
output application_hostname string = springCloudApp.properties.fqdn
```

Note that with **basic** SKU, it seems I don't get the error.

Reply:
done